src/app/views/home.py

```diff
@@ -20,12 +21,27 @@ class Homepage(TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        search_query = self.request.GET.get('search', '')
+        if search_query:
+            delegates_list = []
+            delegates = Delegate.objects.filter(Q(name__icontains=search_query) | Q(address__icontains=search_query))
+            for delegate in delegates:
+                delegate_list = cache.get('app.sql.get_delegate.{}'.format(delegate.slug))
```

Comment:
I actually need to run tests if I still need this cache or not after I did some changed to the sql query

Reply:
👍